tracer/src/Datadog.Trace.Tools.Runner/Utils.cs

```diff
@@ -451,12 +455,33 @@ internal static bool IsAlpine()
             string tracerProfiler64 = string.Empty;
             string tracerProfilerArm64 = null;
             string ldPreload = string.Empty;
+            string devPath = string.Empty;
 
             if (platform == Platform.Windows)
             {
                 tracerProfiler32 = FileExists(Path.Combine(tracerHome, "win-x86", "Datadog.Trace.ClrProfiler.Native.dll"));
                 tracerProfiler64 = FileExists(Path.Combine(tracerHome, "win-x64", "Datadog.Trace.ClrProfiler.Native.dll"));
                 tracerProfilerArm64 = FileExistsOrNull(Path.Combine(tracerHome, "win-ARM64EC", "Datadog.Trace.ClrProfiler.Native.dll"));
+
+                // For full compatibility with .NET Framework we need to either install Datadog.Trace.dll to the GAC or enable Development mode for vstest.console
+                // This is a best effort approach by:
+                //   1. Check if `Datadog.Trace` is installed in the gac using `gacutil`, if not, we try to install it.
+                //   2. If that doesn't work we try to locate `vstest.console.exe.config` to enable debug mode on this
+                //      command so we can inject the DevPath environment variable
+                if (enableGacInstallation)
+                {
+                    if (!EnsureDatadogTraceIsInTheGac(tracerHome) && EnsureNETFrameworkVSTestConsoleDevPathSupport())
+                    {
+                        devPath = Path.Combine(tracerHome, "net461");
+                    }
+                }
+                else
+                {
+                    if (EnsureNETFrameworkVSTestConsoleDevPathSupport())
```

Comment:
I believe this will result in setting DEVPATH whenever `dd-trace run` is used, not just `dd-trace ci run`?

Reply:
Right, changed in https://github.com/DataDog/dd-trace-dotnet/pull/5129/commits/1abe99b180ab9c2bb3986c8c4af85c78ab0ae4bf